core/src/main/codegen/templates/Parser.jj

```diff
@@ -6126,6 +6126,7 @@ SqlPostfixOperator PostfixRowOperator() :
 |   < ARRAY_MAX_CARDINALITY: "ARRAY_MAX_CARDINALITY" >
 |   < AS: "AS" >
 |   < ASC: "ASC" >
+|   < ASCII: "ASCII" >
```

Comment:
Can you also address [Julian's comment](https://issues.apache.org/jira/browse/CALCITE-2599?focusedCommentId=16788417&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16788417) in JIRA :  

> I don't think this needs a parser change. Just add a function.

Reply:
Updated.